src/compute-pr-actions.ts

```diff
@@ -284,10 +286,10 @@ export function process(prInfo: BotNotFail,
     label("Edits Infrastructure", info.editsInfra);
     label("Edits multiple packages", info.hasMultiplePackages);
     label("Author is Owner", info.authorIsOwner);
-    label("No Other Owners", !info.hasNewPackages && info.noOtherOwners);
+    label("No Other Owners", info.hasEditedPackages && info.noOtherOwners);
     label("Too Many Owners", info.tooManyOwners);
     label("Self Merge", info.canBeSelfMerged);
-    label("Config Edit", !info.hasNewPackages && info.editsConfig);
+    label("Config Edit", info.hasEditedPackages && info.editsConfig);

```

Comment:
Beyond the scope of the original change but maybe?
```suggestion
    label("Config Edit", info.editsConfig);